SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetConnection.cpp

```diff
@@ -30,10 +30,23 @@ USpatialNetConnection::USpatialNetConnection(const FObjectInitializer& ObjectIni
 void USpatialNetConnection::BeginDestroy()
 {
 	DisableHeartbeat();
-
+	
 	Super::BeginDestroy();
 }
 
+void USpatialNetConnection::CleanUp()
+{
+	if (USpatialNetDriver* SpatialNetDriver = Cast<USpatialNetDriver>(Driver))
+	{
+		if (!WorkerAttribute.IsEmpty())
+		{
+			SpatialNetDriver->CleanUpConnection(this);
```

Comment:
Can we cleanup the connection regardless of the worker attribute? Seems a little arbitrary otherwise.

Reply:
I had a test that's throwing an error because it calls cleanup on a connection whose WorkerAttribute is empty. Apparently cleanup can be called on garbage collection, so my guess is that some destruction logic happened, WorkerAttribute was cleared, and cleanup is called again after, which throws an error.
I have a similar check on construction, and it does not log any error.

Reply:
Actually, maybe it's just the SpatialOS connection